SlicerCART/src/label_config.yml

```diff
@@ -27,3 +27,17 @@ labels:
   name: test label
   upper_bound_HU: 120
   value: 4
+- color_b: 30
+  color_g: 240
+  color_r: 90
+  lower_bound_HU: 23
+  name: hurray
+  upper_bound_HU: 234
+  value: 5
+- color_b: 30
+  color_g: 20
+  color_r: 10
+  lower_bound_HU: 45
+  name: should work
```

Comment:
default names should not have space)(s) if composed name (e.g should work vs should_work, the second is better)